packages/tools/devtools/devtools/src/test/types/validateDevtoolsPrevious.generated.ts

```diff
@@ -45,6 +45,30 @@ declare function use_old_InterfaceDeclaration_ContainerDevtoolsProps(
 use_old_InterfaceDeclaration_ContainerDevtoolsProps(
     get_current_InterfaceDeclaration_ContainerDevtoolsProps());
 
+/*
+* Validate forward compat by using old type in place of current type
+* If breaking change required, add in package.json under typeValidation.broken:
+* "TypeAliasDeclaration_ContainerKey": {"forwardCompat": false}
+*/
+declare function get_old_TypeAliasDeclaration_ContainerKey():
```

Comment:
This is very strange. `ContainerKey` is a simple type alias of `string`, and has not changed since it was added. I don't know why the type-tests would be hitting this.

Similarly, `HasContainerKey` is a simple interface containing a single member of type `ContainerKey`, which has similarly not changed since it was added.